Index: Pet_Adoption_System/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nspring.datasource.url=jdbc:mysql://localhost:3306/petshelter?useSSL=false&allowPublicKeyRetrieval=true\r\nspring.datasource.username=root\r\nspring.datasource.password=Omar@1998\r\n\r\nspring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect\r\nspring.jpa.hibernate.ddl-auto=update\r\n\r\nserver.port=8081\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/Pet_Adoption_System/src/main/resources/application.properties b/Pet_Adoption_System/src/main/resources/application.properties
--- a/Pet_Adoption_System/src/main/resources/application.properties	(revision e84ae99fa355dc44829622fbf1ba04e64e33a95c)
+++ b/Pet_Adoption_System/src/main/resources/application.properties	(date 1703449842281)
@@ -1,7 +1,7 @@
 
 spring.datasource.url=jdbc:mysql://localhost:3306/petshelter?useSSL=false&allowPublicKeyRetrieval=true
 spring.datasource.username=root
-spring.datasource.password=Omar@1998
+spring.datasource.password=admin
 
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=update
Index: Pet_Adoption_System/src/main/java/Shelter/Pet_Adoption_System/model/Person/Person.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Shelter.Pet_Adoption_System.model.Person;\r\n\r\nimport jakarta.persistence.*;\r\nimport lombok.*;\r\n\r\n@Data\r\n@MappedSuperclass\r\n@NoArgsConstructor\r\n@AllArgsConstructor\r\npublic abstract class Person {\r\n    @Column(name = \"name\")\r\n    protected String name;\r\n\r\n    @Column(name = \"email_address\")\r\n    protected String emailAddress;\r\n\r\n    @Column(name = \"phone_number\")\r\n    protected String phoneNumber;\r\n\r\n    @Column(name = \"password_hash\")\r\n    protected String passwordHash;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Pet_Adoption_System/src/main/java/Shelter/Pet_Adoption_System/model/Person/Person.java b/Pet_Adoption_System/src/main/java/Shelter/Pet_Adoption_System/model/Person/Person.java
--- a/Pet_Adoption_System/src/main/java/Shelter/Pet_Adoption_System/model/Person/Person.java	(revision e84ae99fa355dc44829622fbf1ba04e64e33a95c)
+++ b/Pet_Adoption_System/src/main/java/Shelter/Pet_Adoption_System/model/Person/Person.java	(date 1703449919260)
@@ -6,7 +6,6 @@
 @Data
 @MappedSuperclass
 @NoArgsConstructor
-@AllArgsConstructor
 public abstract class Person {
     @Column(name = "name")
     protected String name;
@@ -19,4 +18,11 @@
 
     @Column(name = "password_hash")
     protected String passwordHash;
+
+    public Person(String name, String emailAddress, String phoneNumber, String passwordHash) {
+        this.name=name;
+        this.emailAddress=emailAddress;
+        this.phoneNumber=phoneNumber;
+        this.passwordHash=passwordHash;
+    }
 }
\ No newline at end of file
